hw/bsp/nucleo-l476rg/src/hal_bsp.c

```diff
@@ -79,7 +93,39 @@ static struct stm32_hal_i2c_cfg i2c_cfg0 = {
     .hic_pin_scl = MCU_GPIO_PORTB(8),     /* PB8 - D15 on CN5 */
     .hic_pin_af = GPIO_AF4_I2C1,
     .hic_10bit = 0,
-    .hic_timingr = 0x10420F13,            /* 100KHz at 8MHz of SysCoreClock */
+    .hic_timingr = 0x00000E14,            /* 100KHz at 4MHz of SysCoreClock */
```

Comment:
Nice, have you used the spreadsheet to calculate this?

Reply:
stm32cube did it for me